src/rmrk2.0.0/tools/consolidator/utils.ts

```diff
@@ -7,25 +7,37 @@ import {
 import { Collection } from "../../classes/collection";
 import { IConsolidatorAdapter } from "./adapters/types";
 import { Base } from "../../classes/base";
+import { List } from "../../classes/list";
 import { changeIssuerInteraction } from "./interactions/changeIssuer";
 import { ChangeIssuer } from "../../classes/changeissuer";
 import { Remark } from "./remark";
 import { OP_TYPES } from "../constants";
 
-export const isNftTransferable = (nft: NFT, remark: Remark) => {
-  return (
-    nft.transferable === 1 ||
-    (nft.transferable < 0 && nft.block - nft.transferable >= remark.block) ||
-    (nft.transferable > 1 && remark.block >= nft.transferable)
-  );
+export const isNftTransferable = (nft: NFT, remark: Remark, opType: OP_TYPES) => {
+  //check transferability if not buy or delist interactions
+  if (opType === "LIST" && List.fromRemark(remark.remark).price !== 0 || opType !== "BUY") {
```

Comment:
I would instead move this logic to list interaction. 

```
if (listEntity.price !== BigInt(0)) {
  validateTransferability(nft, remark, OP_TYPES.LIST);
}
```

Reply:
Should i do the same for buy or would you rather have the nfts not be buyable after the transferable block passed?
```
if (nft.transferable >= 0) {
  validateTransferability(nft, remark, OP_TYPES.BUY);
}
```

Reply:
Not buyable IMO

Reply:
Wouldn't that make it seem like a scam with it being listed but not buyable?

Reply:
A workaround might be delisting it on failed buy if listed after the blocknumber passed but i'm not sure how good it is that this'd require a failed buy interaction

Reply:
nah I think we can just handle it on UI if really needed

Reply:
Ok